src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -273,7 +271,7 @@ namespace FanCoilUnits {
 
         auto &FanCoil(state.dataFanCoilUnits->FanCoil);
         auto &ErrorsFound(state.dataFanCoilUnits->ErrorsFound);
-        auto errFlag(state.dataFanCoilUnits->errFlag);
+        bool errFlag(state.dataFanCoilUnits->errFlag); // What is this trying to do?
```

Comment:
bool &ErrorsFound, no need to comment on auto &FanCoil for this branch effort. Re: errFlag, it should? get set back to state if not acted on directly in this function. Doesn't seem it should be in state but I didn't look.